### DISCUSSION TITLE
Problema tras instalar Practico versión 23.6 en Laragon 6.0

### DISCUSSION THREAD

**Discussion by jtoloza3** [unix4you2/practico#48] [Q&A] [answered]
Hola Jhon!, hemos conocido tu Framework en la Secretaría TIC de la Alcaldia Barrancabermeja, Santander. Felicitaciones por tu inmenso trabajo. Tu framework nos resulta muy interesante y lo encontramos por Datos Abiertos Colombia (https://herramientas.datos.gov.co/usos/framework-practico-para-desarrollo-de-aplicaciones). Queremos probarlo y hemos descargado la versión 23.6;  aparentemente todo quedo bien durante la instalación pero, luego de la instalación no podemos ver nada, sólo una pantalla en blanco del navegador. No sabemos aún por qué.

Estams trabajando con Servidor Laragon 6.0 versiones: 
PHP: 8.1 , 8.3 
MySQL: 8.0.30 

Durante la instalación no se genera problema alguno y se crean todas tablas core_ 

Pero luego no sabemos que hacer.

Agradecemos nos puedas orientar.

**Comment by unix4you2** [comment, marked as answer]
Saludos @jtoloza3 

Me alegra que hayan visto la herramienta y la encuentren de utilidad.   Como verás ya muchas instituciones de gobierno y privada la usan hace varios años.

Gracias por los datos de instalación que provees, ahí de entrada veo que es por la versión de PHP.  Deberías estar en una máxima de 7.4.   Las tablas de compatibilidad reportadas por varios usuarios las encuentras en este enlace https://www.practico.org/inicio/detalles  en el vínculo de "Tablas de compatibilidad".

Importante, la versión PHP 8+ se espera liberar en soporte para diciembre de este año.   Si necesitas cualquier apoyo puedes escribirme inicialmente a correo que no es secreto para nadie unix4you2@gmail.com y coordinamos alguna conferencia por fuera de mis horarios laborales para que sepan sacar todo el provecho a la herramienta y tengan pasos iniciales.

Un abrazo,

**Comment by unix4you2** [comment]
Saludos nuevamente @jtoloza3 

Directamente desde Laragon tal vez este post en su repo pueda ser de ayuda para ti para mover de versión 8 a 7.4:  https://github.com/leokhoa/laragon/discussions/634
